jfx/build.gradle

```diff
@@ -0,0 +1,45 @@
+plugins {
+    id 'java-library'
+    id 'application'
+    id 'distribution'
+    id 'org.openjfx.javafxplugin' version '0.0.10'
+}
+
+repositories {
+    mavenCentral()
+}
+
+apply from: '../buildSrc/misq-version.gradle'
+
+application {
+    project.mainClassName = 'network.misq.jfx.ApplicationMain'
```

Comment:
I think its better to rename to JfxMain to have it consistent with convention used elsewhere, also to not occupy the ApplicationMain class name as that might be used for the application module (or we use Desktop?)

Reply:
Resolved by commit cd3efe5b7d4fdd851f7870835ccc5eb9f90e11a0.